ui/admin/tests/acceptance/host-catalogs/host-sets/hosts-test.js

```diff
@@ -217,26 +229,34 @@ module('Acceptance | host-catalogs | host-sets | hosts', function (hooks) {
   test('visiting host creation from a host set', async function (assert) {
     await visit(urls.createAndAddHost);
     await a11yAudit();
+
     assert.strictEqual(currentURL(), urls.createAndAddHost);
   });
 
   test('create and add host to host set', async function (assert) {
     instances.hostSet.update({ hostIds: [] });
-    await visit(urls.hostSet);
-    assert.strictEqual(findAll('tbody tr').length, 0);
-    await click(MANAGE_DROPDOWN_SELECTOR);
-    await click(CREATE_AND_ADD_HOSTS_SELECTOR);
+    await visit(urls.hostSetHosts);
+
+    assert.dom(commonSelectors.TABLE_ROWS).doesNotExist();
+
+    await click(selectors.MANAGE_DROPDOWN_HOST_SETS);
+    await click(selectors.MANAGE_DROPDOWN_HOST_SETS_CREATE_AND_ADD);
+
     assert.strictEqual(currentURL(), urls.createAndAddHost);
-    await fillIn('[name="name"]', 'Test Name');
-    await fillIn('[name="description"]', 'description');
-    await click('form [type="submit"]:not(:disabled)');
+
+    await fillIn(commonSelectors.FIELD_NAME, 'Test Name');
```

Comment:
nonblocker- noticed we have FIELD_NAME_VALUE and FIELD_DESCRIPTION_VALUE in commonSelectors. Did we want to use those here?